### DISCUSSION TITLE
FOQUS/ HYSYS

### DISCUSSION THREAD

**Discussion by MDCA960418** [CCSI-Toolset/FOQUS#989] [General]
Can FOQUS be connected to Aspen HYSYS?

**Comment by anujad95** [comment]
Currently FOQUS can only be connected to Aspen Plus and Aspen Custom Modeler, as supported by SimSinter.   We don't support connections to HYSYS.

**Reply by MDCA960418** [reply]
Will this connection be included in the future?

**Reply by ksbeattie** [reply]
Unfortunately, we don't have any plans to support it in the future.
